.github/workflows/rust_test.yml

```diff
@@ -38,16 +38,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: check out hyperdrive
```

Comment:
if this works then you can remove the code above
```
      - name: Fetch latest release tag from Hyperdrive
        run: ./scripts/fetch-latest-tag.sh delvtech hyperdrive
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
This is still needed to set the `HYPERDRIVE_REF` environment variable to the latest release below.